internal/manifests/manifests.go

```diff
@@ -327,7 +338,8 @@ func GetManifestObjectName(clusterID strfmt.UUID, fileName string) string {
 
 // GetManifestObjectName returns the manifest object name as stored in S3
 func GetManifestMetadataObjectName(clusterID strfmt.UUID, fileName string, manifestSource string) string {
-	return filepath.Join(string(clusterID), constants.ManifestMetadataFolder, fileName, manifestSource)
+	objectName := filepath.Join(string(clusterID), constants.ManifestMetadataFolder, fileName, manifestSource)
+	return objectName
```

Comment:
Why was this changed?

Reply:
Mainly debugging, will revert it when I commit.